data/defaults.csv

```diff
@@ -0,0 +1,6 @@
+env_setup_commands,pip install --upgrade pip<19
+env_setup_commands,poetry install
+env_setup_commands,pre-commit install
+self_check_commands,pre-commit run -a
+run_tests_commands,python -m pytest
```

Comment:
Should this be `poetry run pytest` since we're already setting up `poetry` in the defaults?

Reply:
all of these commands will be installed _within the project venv_, thus _everything_ it calls is calling venv commands, and the *only* time we need to explicitly call `poetry` is when we need it to do something. `poetry run` is always extraneous.